nextcord/types/activity.py

```diff
@@ -26,6 +27,8 @@
 
 from typing import List, Literal, Optional, TypedDict
```

Comment:
Wouldn't it make sense to use `TypedDict` from `typing_extensions`? Because I don't think the older `TypedDict` can handle `Required` and `NotRequired`

Reply:
All the checks pass, so it seems like it's fine either way

Reply:
Ah yeah that does make sense